sbws/core/scanner.py

```diff
@@ -177,10 +177,9 @@ def measure_relay(args, conf, destinations, cb, rl, relay):
     # exit, then pick a non-exit. Otherwise pick an exit.
     helper = None
     circ_fps = None
-    if relay.can_exit_to(dest.hostname, dest.port) and \
-            relay not in rl.bad_exits:
+    if relay not in rl.bad_exits:
```

Comment:
BadExit is a flag that means an exit has been caught doing bad things (like injecting malicious JavaScript) and that clients should not use it for exiting. 99.9% of relays do not have this flag. This is not what you want to be checking here.

Reply:
i was trying to modify your code as minimum as possible